Update CHANGELOG.md (#249)
ionic-team/cordova-plugin-ionic: CHANGELOG.md

Changelog
======
## 5.5.1
* Updated iOS live update logic to properly handle non-200 status code responses from the update server.

## 5.5.0
* Removed no longer used cordova-plugin-whitelist dependency
* Updated gradle dependencies to use implementation rather than compile

## 5.4.8

> * Fixed background update mode to handle scenario in which a binary update happens after the download of a live update but before the update is applied

Reviewer: ```suggestion
* Fixed background update mode to handle a scenario in which a live update for an older binary is applied to a new binary
```

I think this better represents the problem, but I could be wrong.
